clients/drcachesim/tests/offline-trim-instr.templatex

```diff
@@ -0,0 +1,33 @@
+Output 292 entries from 329 entries.
+Output format:
+<--record#-> <--instr#->: <Wrkld.Tid> <record details>
+------------------------------------------------------------
+           1           0: W0.T1992554 <marker: version 6>
+           2           0: W0.T1992554 <marker: filetype 0xe40>
+           3           0: W0.T1992554 <marker: cache line size 64>
+           4           0: W0.T1992554 <marker: chunk instruction count 20>
+           5           0: W0.T1992554 <marker: page size 4096>
+           6           0: W0.T1992554 <marker: timestamp 13352268558646106>
+           7           0: W0.T1992554 <marker: W0.T1992554 on core 10>
+           8           0: W0.T1992554 <marker: system call 1>
+           9           0: W0.T1992554 <marker: maybe-blocking system call>
+          10           0: W0.T1992554 <marker: timestamp 13352268558646120>
+.*
+         211         112: W0.T1992554 <marker: timestamp 13352268558646661>
+         212         112: W0.T1992554 <marker: W0.T1992554 on core 10>
+         213         113: W0.T1992554 ifetch       4 byte\(s\) @ 0x0000000000401028 48 83 eb 01          sub    \$0x0000000000000001 %rbx -> %rbx
+         214         114: W0.T1992554 ifetch       4 byte\(s\) @ 0x000000000040102c 48 83 fb 00          cmp    %rbx \$0x0000000000000000
+         215         115: W0.T1992554 ifetch       2 byte\(s\) @ 0x0000000000401030 75 d9                jnz    \$0x000000000040100b \(taken\)
+         216         116: W0.T1992554 ifetch       7 byte\(s\) @ 0x000000000040100b 48 c7 c7 01 00 00 00 mov    \$0x0000000000000001 -> %rdi
+         217         117: W0.T1992554 ifetch       8 byte\(s\) @ 0x0000000000401012 48 8d 34 25 00 20 40 lea    0x00402000 -> %rsi
+         217         117: W0.T1992554                                             00
+         218         118: W0.T1992554 ifetch       7 byte\(s\) @ 0x000000000040101a 48 c7 c2 0d 00 00 00 mov    \$0x000000000000000d -> %rdx
+         219         119: W0.T1992554 ifetch       5 byte\(s\) @ 0x0000000000401021 b8 01 00 00 00       mov    \$0x00000001 -> %eax
+         220         120: W0.T1992554 ifetch       2 byte\(s\) @ 0x0000000000401026 0f 05                syscall  -> %rcx %r11
```

Comment:
This test passes before 7 and after 120, right?  But wouldn't we end up with ~113 (see fencepost comments) and not 120 instrs, so we would not see an ordinal as high as 120?